### ISSUE TITLE
[feedback]  It does not have a function to send information about program errors

### ISSUE THREAD

**Issue by pdehaan** [mozilla/fx-private-relay#652]
via https://addons.mozilla.org/en-US/firefox/addon/private-relay/reviews/1587791/

Original:

> No tiene una función para enviar información sobre errores del programa.

Translation:

> It does not have a function to send information about program errors.

**Comment by zanona** [comment]
Yes. It would also be great to be able to report a site which emails are, apparently, being blocked.

The [FAQ](https://relay.firefox.com/faq) mentions:
> __Why won’t a site accept my Relay alias?__
> ...
> If you are not able to use a Relay alias, [please let us know](https://addons.mozilla.org/firefox/addon/private-relay/). 

However, that simply links to the add-on page?
